selfdrive/car/interfaces.py

```diff
@@ -109,9 +111,22 @@ def get_std_params(candidate, fingerprint):
 
   @staticmethod
   def get_torque_params(candidate, default=float('NaN')):
+    with open(TORQUE_SUBSTITUTE_PATH) as f:
```

Comment:
this should also enforce that each car is only defined once

Reply:
willdo